apps/scan/backend/src/app_scan.test.ts

```diff
@@ -1200,3 +1212,58 @@ test('stops completely if plustekctl freezes and cant be killed', async () => {
     error: 'paper_status_timed_out',
   });
 });
+
+test('write scanner report data to card', async () => {
```

Comment:
Moved this test from `app_config.test.ts` to `app_scan.test.ts` since it fits better under the latter